sdk/storage/azfile/sas/query_params.go

```diff
@@ -17,7 +17,7 @@ import (
 // timeFormat represents the format of a SAS start or expiry time. Use it when formatting/parsing a time.Time.
 const (
 	timeFormat         = "2006-01-02T15:04:05Z" // "2017-07-27T00:00:00Z" // ISO 8601
-	snapshotTimeFormat = "2006-01-02T15:04:05.0000000Z07:00"
+	SnapshotTimeFormat = "2006-01-02T15:04:05.0000000Z07:00"
```

Comment:
Is this variable exposed from this package ?

Reply:
This variable is used by the share client's `GetSASURL()` method to parse the share snapshot time to its appropriate format.